app/controllers/users/webauthn_setup_controller.rb

```diff
@@ -156,6 +159,10 @@ def process_valid_webauthn(form)
         platform_authenticator: form.platform_authenticator?,
```

Comment:
Hm, I'm not sure I realized we had this other `multi_factor_auth_added_webauthn` event, which sounds somewhat redundant now? Or at least, it seems like we could absorb that into the new submitted event?

Reply:
It can be absorbed because that information plus the error gives us more information.

Reply:
Could we keep the payload of webauthn_setup_visit the same? That is, leaving the lines:

```
      properties = result.to_h.merge(analytics_properties)
      analytics.webauthn_setup_visit(**properties)
```

This will make upcoming analysis easier, as the history of the event would not change.

Reply:
I don't think it'll be sustainable to maintain backward compatibility long-term. For the upcoming analysis, would it be enough to use either a timeframe from exclusively before or exclusively after the changes being introduced here?

Reply:
It sounds like exclusively before would be the way to go on this because I am concerned about tracking two very similar events with very similar information. @jc-gsa let me know if I am off-base.

Reply:
The event is enhanced by the extra information. Visually, the code is cleaner as well. Leaving the original payload seems like a win-win.

For upcoming analysis, I can write down the date the payload changed and then adapt analysis. Leaving the code the same has benefits and seemingly no cost.

Reply:
I hear what you're saying, but I am going to keep what I have as-is because `**properties` still has the success and error values that I do not want to send through to the event.

```
[1] pry(#<Users::WebauthnSetupController>)> result
=> #<FormResponse:0x0000000111068080
16:24:17 web.1    |  @error_details={},
16:24:17 web.1    |  @errors={},
16:24:17 web.1    |  @extra={:platform_authenticator=>true, :enabled_mfa_methods_count=>0},
16:24:17 web.1    |  @serialize_error_details_only=false,
16:24:17 web.1    |  @success=true>
```